source/shared/core_sqlsrv.h

```diff
@@ -1055,8 +1055,13 @@ struct stmt_option;
 // This holds the various details of column encryption. 
 struct col_encryption_option {
     bool            enabled;            // column encryption enabled, false by default
+    SQLINTEGER      akv_mode;
+    zval_auto_ptr   akv_auth;
+    zval_auto_ptr   akv_id;
```

Comment:
Is akv_auth is still needed now that you have akv_mode? You can initialize akv_mode to an invalid value ?